allensdk/brain_observatory/behavior/data_objects/metadata/subject_metadata/full_genotype.py

```diff
@@ -14,7 +14,7 @@ class FullGenotype(DataObject, LimsReadableInterface, JsonReadableInterface,
                    NwbReadableInterface):
     """the name of the subject's genotype"""
     def __init__(self, full_genotype: str):
-        super().__init__(name="full_genotype", value=full_genotype)
+        super().__init__(name="full_genotype", value=str(full_genotype))
```

Comment:
So I understand, you are coercing `None` to the string `'None'` which will later fail a check to see if the genotype is valid?

Could you please make the type for `full_genotype` to be `Optional[str]` since it can be null?

Reply:
Done